<!doctype html>
<html>

  <head>
    <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');</script>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <title>Laurie Prelot</title>

    <link rel="stylesheet" href="stylesheets/styles.css">
    <link rel="stylesheet" href="stylesheets/pygment_trac.css">
    <link rel="shortcut icon" type="image/png" href="./images/favicon.png"/>
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">

  </head>
  <body>
    <div class="wrapper">

      <section>

<p><img style="padding: 0 15px; float: left;" src="./images/portrait_VF_2020_sq.jpg" height=190 width=190/> 
  <font size=5>Laurie Prelot, PhD Candidate</font> <br>
        <font size=2>PhD Candidate</font> <br>
	<a href=https://inf.ethz.ch/de/">ETH Zürich, Department of Computer Science</a><br>
        <a href="https://ml.inf.ethz.ch/"> Institute for Machine Learning</a> <br>
        <a href="https://bmi.inf.ethz.ch/">Biomedical Informatics group</a> <br>

        Email: laurie.prelot at inf.ethz.ch <br></p>
        <a href="https://github.com/lthp" target="_blank" style="padding-left: 20px"><img src="./images/github.svg" height="20px" width="20px"></a>
        <a href="https://ch.linkedin.com/in/laurie-pr%C3%A9lot" target="_blank" style="padding-left: 20px"><img src="./images/linkedin.svg" height="20px" width="20px"></a>
        </p>

<br>
<br>

<!-- <h1> Bio </h1> -->
<p align='justify'>I am PhD candidate at <a href="https://ethz.ch/en.html">ETH Zürich</a> under the supervision of <a href="https://bmi.inf.ethz.ch/people/person/gunnar-raetsch/">Gunnar Rätsch</a> member of the <a href="https://bmi.inf.ethz.ch/">Biomedical Informatics group</a> 
</p>
I work on the in silico prediction of tumor neoepitopes and their relationship with the immune compartment for immunotherapy. I am currently developing a software tool, to integrate RNA and DNA alterations into a graph representation and generate candidate neoepitopes. I also work in collaboration with <a href="https://nellore.bio/"> A. Nellore</a>- <a href="https://www.ohsu.edu/school-of-medicine/radiation-medicine/thompson-lab">R. Thompson Lab</a> at OHSU to ensure algorithmic reproducibility of the neoepitoes generation and differential filtering. I also aim at developing novel approaches to search rare neoepitope candidates in proteomic databases and develop a predictive model of validation. I plan to use the software tool to make biological discoveries. The former, combined with data analytics techniques will allow the stratification of patients and the prediction of immunotherapy outcomes in relationship with their omics status.  
<p align='justify'>
I did 
</p>



<h1> Publications</h1> 

<ol>

<!-- remove first <br> tag, otherwise there's a weird space -->


<li> <b>The paper presenting the software tool is coming out soon!</b>
<br>
<b>You can checkout the git repository below:</b>
<br>

<br>
[<a href="https://github.com/ratschlab/immunopepper">code</a>]
</li>


<br>
<li> <b>Heterogeneity in old fibroblasts is linked to variability in reprogramming and wound healing</b>
<br>
<b>Laurie Prelot</b>, Salah Mahmoudi, Elena Mancini, Lucy Xu, Alessandra Moore, Fereshteh Jahanbani, Katja Hebestreit, Rajini Srinivasan, Xiyan Li, Keerthana Devarajan, Laurie Prélot, Cheen Euong Ang, Yohei Shibuya, Bérénice A. Benayoun, Anne Lynn S. Chang, Marius Wernig, Joanna Wysocka, Michael T. Longaker, Michael P. Snyder & Anne Brunet 
<br>
Nature, 2019
<br>
[<a href="https://www.nature.com/articles/s41586-019-1658-5">paper</a>]
</li>

<h1> Classes Taken during the PhD </h1>
<ul>
        <li> <i>Fall 2019</i> Deep Learning 263-3210-00L
  <br>
 <li> <i>Fall 2020</i> Probabilistic Artificial Intelligence 263-5210-00L 
  <br>
 <li> <i>Summer 2019</i> Selected applicant to follow the Machine Summer School 2019, Skoltech, Moscow, Russia
</li>
</ul>


<h1> Teaching </h1>
<ul>
	<li> <i>Spring 2020</i> Teaching assistant for Computional Challenges in Medical Genomics Seminar 261-5113-00L
    <br>
    <li> <i>Spring 2019</i> Teaching assistant for Machine Learning Seminar 252-4811-00L
    <br>
    <li> <i>Spring 2019, 2020 and 2021</i> Introduction to Machine Learning Course 252-0220-00L (800 - 1200 students enrolled)
    <br>
    <li> <i>Fall 2019, 2020</i> Computational Biomedicine Course 261-5100-00L 

<h1> Extracurricular activities </h1>
<ul>
<h2> Director at NEO Network </h2>
The <a href="https://www.neonetwork.ch/">NEO Network</a> is a Swiss non-profit Think thank and network at the largest Swiss Universities that aim to explore the challenges brought by technology. From 2019 to 2020 I was director and I fully organized the discussion panel "Data against Covid19. The recording of the panel can be viewed <a href="https://www.neonetwork.ch/codevscovid19/">here</a>

<ul>
<h2> Ambassador for Ubuntu Mundo </h2>
<a href="https://ubuntu-mundo.org/our-approach-and-impact/">UBUNTU Mundo</a> is a NGO operating since June 2016 in South Africa. The NGO aims at feeding children through the creation of community managed gardens and kitchen. In 2018, I Created a team to raise funds at the 20 KM de Lausanne event

<h2> Developed a Pre-School in Mexico </h2>
In 2013, I promoted the school in local communities, I defined pedagogical objectves, designed the semester’s curriculum


<h1> Short CV </h1>
<ul>
<li> <i>Oct. 2018-Present</i> PhD Candidate in Computer Science ETH
<li> <i>2016-2018</i>: Master of Science in Life Science and Technology <a href="https://www.epfl.ch/education/master/fr/programmes/ingenierie-sciences-vivant/"> at Ecole Polytechnique Federale de Lausanne (EPFL)</a> with focus on Bioinformatics and Data Analysis
<li> <i>2015-2016</i>: Master and Bachelor of Science at <a href="http://www.ens-lyon.fr/formation/offre-de-formation/master/biosciences-sante">Ecole normale Superieure de Lyon in Biology</a> in Biology with a focus in Modelling
<li> <i>2012-2014</i>: Classe Preparatoire aux Grandes Ecoles at Lycee Henri IV<a href="https://www.enseignementsup-recherche.gouv.fr/cid20182/classes-preparatoires-aux-grandes-ecoles-c.p.g.e.html#Les_classes%20pr%C3%A9paratoires%20scientifiques"> CPGE (BCPST)</a> (Undergraduate Physics, Chemistry, Biology, Mathematics)
</ul>
<p align='justify'>
I completed internships at Stanford University under the supervision of <a ref="https://web.stanford.edu/group/brunet/">Anne Brunet</a> (Aging); at EPFL under the supervision of <a href="https://people.epfl.ch/jacques.fellay">Jacques Fellay</a> (Human Genomics) and at <a ref="https://www.lhs-sa.ch/">Lambda Health System</a> (Clinical assays analysis), a Swiss MedTech start-up. I completed my Master Thesis at Imperial College London in the team of <a href="http://www.imperial.ac.uk/pancreatic-islet-diabetes-consortium/research/genetics-genomics-and-personalised-medicine-in-diabetes/">Inga Prokopenko</a>. There, I investigated glyceamic traits prediction from multi-omics data  in diabetic patients.
</p>
<br>

<h1> Awards </h1>
<ul>
<li> <i>2014-2018</i> Awarded a four year elite study grant by the French Government


      </section>

        <div>
  <p> <i>Template Design by <a href="http://aditya-grover.github.io"> Aditya Grover</a>. </i></p>
</div>


    </div>


        
  </body>



</html>
